packages/vitepress-plugin-og-image/src/utils/svg/render.ts

```diff
@@ -0,0 +1,65 @@
+import type { Buffer } from 'node:buffer'
+import { Resvg } from '@resvg/resvg-js'
+import { removeEmoji } from '../emoji'
+import { escape } from './escape'
+
+export function renderSVGWithTemplate(siteName: string, siteDescription: string, title: string, category: string, ogTemplate: string): string {
```

Comment:
Maybe use `renderSVG` better? Than use paragraph comments to explain?

Reply:
Ok.

Reply:
While it's good to have you here to help me on improving the naming of things. I appreciate that. But, I do still want to point out that functions should be able to express their job, meaning, results and even parameters instead of simple words, especially by comparing to overload functions where you may have different arguments of functions but their name is the same, I personally think it's hard for developers to refactor and understand which function is for what.